### PR TITLE
Add PF4/React components for Vertical Navbar (2nd round)

### PR DESCRIPTION
**Pull request by josemigallas**
> ⚠️Draft until #1019 is merged

**What this PR does / why we need it**:

Once PF React no longer embed stylesheets inline, it is possible to use PF React components without destroying our own styles.

Reimplementing #954

**Which issue(s) this PR fixes** 

https://issues.jboss.org/browse/THREESCALE-2808

**Verification steps** 

Verify styles look like they should.

### CODE REVIEW

File: Gemfile.base [hunk 1]
```diff
@@ -76,7 +76,7 @@ gem 'cancancan', '~> 1.7.1'
 gem 'formtastic', '~> 1.2.4'
 gem 'rmagick', '~> 2.15.3', require: false
 gem 'gruff', '~>0.3', require: false
-gem 'bugsnag', '~> 6.0'
+gem 'bugsnag', '~> 6.11'
```
Comment (Martouta) [inline Gemfile.base]: Imo this should be in an independent PR 🤔

**Comment by josemigallas** [comment]
Closing in favor of #1525
